scripts/install_moodle.sh

```diff
@@ -883,6 +883,9 @@ EOF
    #else # mssql. TODO It's missed earlier! Complete this!
    fi
 
+   # setting user sessions to DB
+   sed -i "23 a \$CFG->session_handler_class = '\\\core\\session\\\database';" /moodle/html/moodle/config.php
```

Comment:
We did try this as well, but it didn't give any noticeable perf improvement. Still we can have this, but I'd like this to be grouped together with code where the `/moodle/html/moodle/config.php` is updated, like line 860. I think the 3 PRs could have been just one.